data/film-tv/list-of-Disney-featurepfilms.json

```diff
@@ -0,0 +1,1618 @@
+[
+  {
```

Comment:
I believe the standard format for corpora is to start with an object and then include the list as a named array. It's also good practice to include "description" and "source" properties for more info about the data and where it was compiled from!

```javascript
{
  "description": "____________",
  "source": "____________",
  "films": [ ]
}
```